orgs/vkt.json

```diff
@@ -14,30 +14,35 @@
 
   "urls": {
     "privacyDeclarationUrl": {
-      "default": "https://vkt.no"
+      "default": "https://www.vkt.no/vilkaar-og-personvern/personvern/"
```

Comment:
Should be the tilgjengelighetserklæring url, no?

Reply:
Yes, I looks like I mixed them. Will fix now.